### ISSUE TITLE
[BUG] In some low-priced coin stocks, the crosshair moves differently.

### ISSUE THREAD

**Issue by big100** [highfestiva/finplot#403]
### Code to reproduce

```
import sys 
from PySide2.QtWidgets import *
from PySide2.QtCore import *
import finplot as fplt
import pyupbit

from PySide2.QtTest import QTest
class MyWindow(QMainWindow):
    def __init__(self):
        super().__init__()
        widget= QGraphicsView()
        layout = QGridLayout(widget)
        self.setCentralWidget(widget)
        self.resize(800, 300)
        ax = fplt.create_plot(init_zoom_periods=100)
        widget.axs = [ax]
        layout.addWidget(ax.vb.win)

        df = pyupbit.get_ohlcv("KRW-BTT", interval="minutes1") # there is a problem
        # df = pyupbit.get_ohlcv("KRW-SHIB", interval="minutes1") # coin stocks with a low price
        # df = pyupbit.get_ohlcv("KRW-XEC", interval="minutes1") # coin stocks with a low price
        
        print(df)
        
        fplt.autoviewrestore()
        plot = fplt.candlestick_ochl(df[['open', 'close', 'high', 'low']], ax=ax)
        
        fplt.show(qt_exec=False)

if __name__ == "__main__":
    app = QApplication(sys.argv)
    window = MyWindow()
    window.show()
    app.exec_(app.exec_())
```

### Describe the bug
On the chart, the mouse does not move to the correct value on the Y axis.

This problem occurs with coins of a certain price.

Problems occur on some charts with lower prices.

Even if the price chart has the same decimal point, there will be a difference according to the price.

It should move in increments of 0.0001.
But it moves in increments of 0.001.

### Expected behavior
It is the order book of the coin.
It should move in increments of 0.0001.

![image](https://user-images.githubusercontent.com/37491415/215339058-aa7365b7-7e54-4b0d-9caa-904c53e58468.png)

#### Screenshots
![image](https://user-images.githubusercontent.com/37491415/215338940-46517d5f-8e96-4fc7-9716-6355c8e49235.png)

#### Reproducible in:

*finplot version*: 1.9.0
*pyqtgraph version*: 0.13.1
*PySide2 version*: 5.15.2.1

**Comment by highfestiva** [comment]
Good catch, fix in b6ec5fd.

**Comment by big100** [comment]
> Good catch, fix in [b6ec5fd](https://github.com/highfestiva/finplot/commit/b6ec5fd4c5274fb068e4731b1ee9d3e6700b43ea).

Thanks for solving it.
